File: Easy/JavaScript/1979. Find Greatest Common Divisor of Array.js
var findGCD = function(nums) {
    nums.sort((a, b) => a - b);
    let min = nums[0]
    let max = nums[nums.length - 1];
    let greatest = 0;

    for(let i = 1; i <= max; i++){
        if(max % i === 0 && min % i === 0)
            greatest = i;
    }

    return greatest;
};

File: Easy/Java/1979. Find Greatest Common Divisor of Array.java
class Solution {
    public int findGCD(int[] nums) {
        Arrays.sort(nums);
        int min = nums[0];
        int max = nums[nums.length - 1];
        int greatest = 0;

        for(int i = 1; i <= max; i++){
            if(max % i == 0 && min % i == 0)
                greatest = i;
        }

        return greatest;
    }
}
